pyproject.toml

```diff
@@ -4,6 +4,7 @@ package-mode = false
 [project]
 name = "auto-archiver-api"
 description = "API wrapper for Bellingcat's Auto Archiver, supports users, groups, sheet and url archives."
+version = "0.1.0"
```

Comment:
https://peps.python.org/pep-0621/#version

Reply:
we are defining the version here https://github.com/bellingcat/auto-archiver-api/blob/eec4c094411615898155f7ee59d4719daf7766ef/app/web/config.py#L1 and that version is served on one of the endpoints, I avoided having this version in the pyptoject.toml to avoid confusion and because it did not seem straightforward to import the version from the code as we were installing this with `poetry install --no-root`. do you know of a way to achieve that? if so, I'm happy to remove it from config.py and bring it here.

Reply:
When I was running pre-commit` in another PR, poetry stated that version was a required field. Maybe we can get past it with saying it's not a package?